docs/.readthedocs.yaml

```diff
@@ -8,7 +8,8 @@ build:
   jobs:
      install:
       - which mamba
-      - cd icechunk-python && maturin build && pip install ../target/wheels/*.whl && cd ../docs
+
+      - cd icechunk-python && maturin build && pip install "$(ls ../target/wheels/*.whl | head -n 1)[docs]" && cd ../docs
```

Comment:
I wonder why moving th einstall to pip is so much faster. I would have suspected that it would be slower?

Reply:
pip is quite good, just can be bad at compiled dependencies.